source/deployment/sso-saml-adfs.md

```diff
@@ -0,0 +1,199 @@
+
+![adfs_11_authorization.PNG](../../source/images/adfs_11_authorization.PNG)
+
+11) In the **Ready to Add Trust** screen, you can review your settings.
+
+![adfs_12_ready_to_add_trust.PNG](../../source/images/adfs_12_ready_to_add_trust.PNG)
+
+12) In the **Finish** screen, select the option **Open the Edit Claim Rules dialog for this relying party trust when the wizard closes**, and click **Close**. You will now exit configuration wizard and a **Claim Rules** editor opens.
+
+![adfs_13_finish_trust.PNG](../../source/images/adfs_13_finish_trust.PNG)
+
+### Creating Claim Rules
+
+13) In the **Issuance Transform Rules** of the **Claim Rules** editor, click the **Add Rule…** button. This action opens an **Add Transform Claim Rule Wizard**.
+
+![adfs_14_claim_rules_editor.PNG](../../source/images/adfs_14_claim_rules_editor.PNG)
+
+14) In the **Choose Rule Type** screen, select **Send LDAP Attributes as Claims** from the drop-down menu, then click **Next**.
+
+![adfs_15_choose_rule_type.PNG](../../source/images/adfs_15_choose_rule_type.PNG)
+
+15) In the **Configure Claim Rule** screen, enter a **Claim Rule Name** of your choice, select **Active Directory** as the **Attribute Store** and do the following:
+  - From the **LDAP Attribute column**, select `E-Mail-Addresses`. From the **Outgoing Claim Type**, type `Email`
+  - From the **LDAP Attribute column**, select `Given-Name`. From the **Outgoing Claim Type**, type `FirstName`
+  - From the **LDAP Attribute column**, select `Surname`. From the **Outgoing Claim Type**, type `LastName`
+  - From the **LDAP Attribute column**, select `SAM-Account-Name`. From the **Outgoing Claim Type**, type `Username`
+
+Then, click **OK** to add the rule.
+
+Note that the entries in the **Outgoing Claim Type** column can be chosen to be something else. They can contain dashes but no spaces. Note that they will be used to map the corresponding fields in Mattermost later.
+
+![adfs_16_configure_claim_rule.PNG](../../source/images/adfs_16_configure_claim_rule.PNG)
+
+16) Create another new rule by clicking the **Add Rule…** button.
+
+17) In the **Choose Rule Type** screen, select **Transform an Incoming Claim** from the drop-down menu, then click **Next**.
+
+![adfs_17_transformation_of_incoming_claim.PNG](../../source/images/adfs_17_transformation_of_incoming_claim.PNG)
+
+18) In the **Configure Claim Rule** screen, enter a **Claim Rule Name** of your choice, then
+  - Select `Name ID` for the **Incoming claim type**
+  - Select `Unspecified` for the **Incoming name ID format**
+  - Select `E-Mail Address` for the **Outgoing claim type**
+
+Moreover, select the **Pass through all claim values** option. Then click **Finish**.
+
+![adfs_18_configure_incoming_claim.PNG](../../source/images/adfs_18_configure_incoming_claim.PNG)
+
+19) Click **OK** to create the claim rule, and **OK** again to finish creating rules.
```

Comment:
Click **Finish** to create the claim rule, and **OK** again to finish creating rules.